app/validators/answers/number_answer_validator.py

```diff
@@ -71,6 +74,42 @@ def validate_value_in_limits(self):
                 self.MAXIMUM_GREATER_THAN_LIMIT, value=max_value, limit=MAX_NUMBER
             )
 
+        if isinstance(answer_source_max_value, dict):
+            source = answer_source_max_value.get("source")
+            answers_with_context = self.questionnaire_schema.numeric_answer_ranges
+            value_source_max_value = (
+                self.questionnaire_schema.get_numeric_value_for_value_source(
+                    source, answer_source_max_value, answers_with_context
+                )
+            )
+            if (
+                isinstance(value_source_max_value, int)
+                and value_source_max_value < MAX_NUMBER
+            ):
+                self.add_error(
+                    self.MAXIMUM_GREATER_THAN_LIMIT,
+                    value=value_source_max_value,
+                    limit=MAX_NUMBER,
+                )
+
+        if isinstance(answer_source_min_value, dict):
+            source = answer_source_min_value.get("source")
+            answers_with_context = self.questionnaire_schema.numeric_answer_ranges
+            value_source_min_value = (
+                self.questionnaire_schema.get_numeric_value_for_value_source(
+                    source, answer_source_min_value, answers_with_context
+                )
+            )
+            if (
+                isinstance(value_source_min_value, int)
+                and value_source_min_value > MIN_NUMBER
```

Comment:
```suggestion
                and value_source_min_value > MIN_NUMBER
```
Doesn't this logic need to be reversed? We want to add an error if the value is less than the minimum limit